openapi/components/schemas/GatewayAccount.yaml

```
@@ -256,6 +256,10 @@ properties:
     description: |-
       Forces Dynamic Currency Conversion (DCC) to the specified currency on each sale.
       To disable forced DCC, leave this field empty.
+  dccForceRounding:
+    type: boolean
+    default: false
```

Perhaps example should be also included?

```suggestion
    default: false
    example: false
```